docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,76 @@ To set these configuration options, create a ``Config`` object with the options
+
+.. _specify_proxies:
+
+Specifying proxy servers
+''''''''''''''''''''''''
+
+You can specify proxy servers to be used for connections when using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary in which each entry maps a protocol to the address and port number of the proxy server for that protocol.
+
+In the following example, a proxy list is set up to use ``proxy.amazon.com``, port 6502 as the proxy for all HTTP requests by default. HTTPS requests use port 2010 on ``proxy.amazon.org`` instead.
+
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_definitions = {
+        'http': 'http://proxy.amazon.com:6502',
+        'https': 'https://proxy.amazon.org:2010'
+    }
+
+    my_config = Config(
+        'region_name': 'us-east-2',
+        'signature_version': 'v4',
+        'proxies': proxy_definitions
+    }
+
+    client = boto3.client('kinesis', config=my_config)
+
+
+.. _configure_proxies:
+
+Configuring proxies
+'''''''''''''''''''
+You can configure how Boto3 uses proxies by specifying the ``proxies_config`` option, which is a dictionary that specifies the values of several proxy options by name.  There are three keys in this dictionary: ``proxy_ca_bundle``, ``proxy_client_cert``, and ``proxy_use_forwarding_for_https``. For more information about these keys, see the `Botocore config reference <https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html#botocore.config.Config>`_.
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_definitions = {
+        'http': 'http://proxy.amazon.com:6502'
+    }
+
+    my_config = Config(
+        'region_name': 'us-east-2',
+        'signature_version': 'v4',
+        'proxies': proxy_definitions,
+        'proxies_config': {
+            'proxy_client_cert': '/path/of/certificate'
+        }
+    }
+
+    client = boto3.client('kinesis', config=my_config)
+
+With the addition of the ``proxies_config`` option shown here, the proxy will use the specified certificate file for authentication.
+
+
 Using environment variables 
 ---------------------------
-Configurations can be set through the use of system-wide environment variables. If set, these configurations are global and will affect all clients created unless explicitly overwritten through the use of a ``Config`` object. 
+You can set configuration settings using system-wide environment variables. These configurations are global and will affect all clients created unless you override them with a ``Config`` object.
+
+.. note::
+    Not all configuration settings can be set using environment variables. The `Using environment variables <https://docs.aws.amazon.com/credref/latest/refdocs/environment-variables.html>`_ chapter of AWS SDKs and Tools Shared Configuration and Credentials provides this information.
```

Comment:
I'm not sure we want to be linking out to that other doc when we already list the Environment Variables we support immediately below. I believe that doc is more likely to be out of date than the one we're currently in.

Reply:
OK.